<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.269">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Modesto">
<meta name="dcterms.date" content="2023-01-16">

<title>R’n’R lab - Mobile Genetic Elements</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<style>html{ scroll-behavior: smooth; }</style>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./figs/rnr_logo.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">R’n’R lab</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-research" role="button" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Research</span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-research">    
        <li>
    <a class="dropdown-item" href="./polb.html">
 <span class="dropdown-text">DNA polymerases</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./mge.html">
 <span class="dropdown-text">Mobile Genetic Elements</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./mda.html">
 <span class="dropdown-text">DNA polymerases in biotechnology</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./tecti.html">
 <span class="dropdown-text">Tectiviruses as a model</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./people.html">
 <span class="menu-text">People</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./funding.html">
 <span class="menu-text">Funding</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./teaching.html">
 <span class="menu-text">Teaching</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./contact.html">
 <span class="menu-text">Contact us</span></a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#mobile-genetic-elements" id="toc-mobile-genetic-elements" class="nav-link active" data-scroll-target="#mobile-genetic-elements">Mobile genetic elements</a></li>
  <li><a href="#self-replicative-mges-or-not" id="toc-self-replicative-mges-or-not" class="nav-link" data-scroll-target="#self-replicative-mges-or-not">Self-replicative MGEs… or not?</a>
  <ul class="collapse">
  <li><a href="#pipolins-our-model-mge" id="toc-pipolins-our-model-mge" class="nav-link" data-scroll-target="#pipolins-our-model-mge">Pipolins, our model MGE</a></li>
  <li><a href="#explorepipolin" id="toc-explorepipolin" class="nav-link" data-scroll-target="#explorepipolin">ExplorePipolin</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Mobile Genetic Elements</h1>
  <div class="quarto-categories">
    <div class="quarto-category">Mobile element</div>
    <div class="quarto-category">Bacteriophage</div>
    <div class="quarto-category">Pipolin</div>
    <div class="quarto-category">Bioinformatics</div>
    <div class="quarto-category">Patogen</div>
  </div>
  </div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Author</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <a href="https://rnrlab.github.io">Modesto</a> 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Biochemistry, UAM
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">January 16, 2023</p>
    </div>
  </div>
  
    <div>
    <div class="quarto-title-meta-heading">Modified</div>
    <div class="quarto-title-meta-contents">
      <p class="date-modified">February 13, 2023</p>
    </div>
  </div>
    
  </div>
  

</header>

<section id="mobile-genetic-elements" class="level1">
<h1>Mobile genetic elements</h1>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
<strong>Acknowledgment</strong>
</div>
</div>
<div class="callout-body-container callout-body">
<p>This brief section belongs to the Introduction of the Master’s Thesis from one of our students, Víctor Mateo-Cáceres (UAM, June 2022).</p>
</div>
</div>
<p><img src="images/ConcreteMessyKob-size_restricted.gif" title="Under construction..." class="img-fluid"></p>
<div class="callout-warning callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Watch out! This page is still being synthesized!
</div>
</div>
<div class="callout-body-container callout-body">

</div>
</div>
<p>Over the course of evolution, genetic material has found diverse ways of mobilization beyond vertical inheritance. A whole spectrum of vehicles, ranging from plasmids to phages, is used by all bacterial species for the horizontal transference of DNA between cells <span class="citation" data-cites="frost2005">Partridge et al. (<a href="#ref-partridge2018" role="doc-biblioref">2018</a>)</span>. These wandering DNA sequences, referred to as Mobile Genetic Elements (MGE), can encode virulence determinants, antimicrobial resistance (AMR) genes, phage defense systems, and other factors that grant their hosts new phenotypic traits <span class="citation" data-cites="khosts">(<a href="#ref-khosts" role="doc-biblioref"><strong>khosts?</strong></a>’2020)</span> . MGE transference is especially relevant in prokaryotes, where specific DNA mobilization systems have been developed to enhance and optimize the acquisition of these elements <span class="citation" data-cites="frost2005">(<a href="#ref-frost2005" role="doc-biblioref">Frost et al. 2005</a>)</span>. In fact, obtaining advantageous features through MGE transference is the main form of short-term adaptation in bacteria and can become crucial for cell survival, as in the case of AMR genes. The concerning increase of multidrug-resistant (MDR) bacteria is a clear example of this last point <span class="citation" data-cites="magiorakos2012">(<a href="#ref-magiorakos2012" role="doc-biblioref">Magiorakos et al. 2012</a>)</span>. Therefore, it is essential to discover and characterize new mechanisms of genetic transference in order to disclose which features allow certain bacterial strains to succeed over the rest, especially the pathogenic ones.</p>
<p>Horizontal genetic exchange mechanisms can be classified into three main groups according to the DNA introduction pathway: transformation, transduction, and conjugation (Figure 1). Transformation is the direct uptake of genetic material from the extracellular medium, which is frequently inefficient and can require activating the competence state <span class="citation" data-cites="carvalho2020">Firth et al. (<a href="#ref-firth2018" role="doc-biblioref">2018</a>)</span>. Transduction refers to the transport of foreign DNA by viral particles <span class="citation" data-cites="humphrey2021">(<a href="#ref-humphrey2021" role="doc-biblioref">Humphrey et al. 2021</a>)</span>. Conjugation, which has been extensively studied in plasmids, involves the transference of DNA from a donor cell to a recipient cell through the Type IV Secretion System (T4SS) <span class="citation" data-cites="smillie2010">(<a href="#ref-smillie2010" role="doc-biblioref">Smillie et al. 2010</a>)</span>.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figs/hgt.png" class="img-fluid figure-img" width="489"></p>
<p></p><figcaption class="figure-caption">Figure 1. Main mechanisms of genetic horizontal transfer. (A) Transformation: direct extracellular DNA uptake by the cell. (B) Transduction: DNA transference through the injection of a phage genetic content. (C) Conjugation: DNA transference through the Type IV Secretion System. T4SS: Type IV Secretion System, T4CP: T4SS coupling protein. Modified with Biorender from Frost et. al (2005)</figcaption><p></p>
</figure>
</div>
<p>Conjugative plasmids typically carry all genes necessary for T4SS configuration plus a T4SS coupling protein (T4CP), which is involved in the T4SS-DNA interaction, and a relaxase, which is a nickase necessary for generating the single-stranded DNA (ssDNA) that will be transferred. In fact, relaxases can be divided into several families according to their sequence and have been proposed as markers for plasmid classification <span class="citation" data-cites="smillie2010 garcillan-barcia2009">(<a href="#ref-smillie2010" role="doc-biblioref">Smillie et al. 2010</a>; <a href="#ref-garcillan-barcia2009" role="doc-biblioref">Garcillan-Barcia, Francia, and Cruz 2009</a>)</span>. Nevertheless, we currently know conjugative plasmids that lack some of the conjugation modules and conjugative plasmids with the capacity of integrating into the genome, also called Integrative and Conjugative Elements (ICEs) <span class="citation" data-cites="botelho2021">(<a href="#ref-botelho2021" role="doc-biblioref">Botelho and Schulenburg 2021</a>)</span>, remarking the high flexibility of this type of MGEs.</p>
</section>
<section id="self-replicative-mges-or-not" class="level1">
<h1>Self-replicative MGEs… or not?</h1>
<p>We are particularly interested in elements encoding enzymes related to DNA replication and repair. Thus, the discovery, some time ago, of a group of MGEs called Polintons <span class="citation" data-cites="kapitonov2006">Kapitonov and Jurka (<a href="#ref-kapitonov2006" role="doc-biblioref">2006</a>)</span>, as they encode their own B-family DNA polymerase. Later, another MGEs group also resulted to contain a PolB, the Casposons <span class="citation" data-cites="krupovic2014">Krupovic et al. (<a href="#ref-krupovic2014" role="doc-biblioref">2014</a>)</span>. This led some authors to define a group of elements that might be self-replicative elements <span class="citation" data-cites="krupovic2016">Krupovic and Koonin (<a href="#ref-krupovic2016" role="doc-biblioref">2016</a>)</span>. However, the role of those PolBs is still unknown.</p>
<section id="pipolins-our-model-mge" class="level2">
<h2 class="anchored" data-anchor-id="pipolins-our-model-mge">Pipolins, our model MGE</h2>
<p>Apart from phages, plasmids, ICEs, and other MGEs that have been in the spotlight for the last years, such as integrons or CRISPR-Cas Associated Transposons (CASTs), one group recently discovered stands out for its large genetic diversity and variability: pipolins. This new superfamily of MGEs comprises elements found in the three major bacterial phyla and even mitochondria <span class="citation" data-cites="redrejo-rodríguez2017">Redrejo-Rodríguez et al. (<a href="#ref-redrejo-rodríguez2017" role="doc-biblioref">2017</a>)</span>. The only gene shared by all pipolins encodes for a replicative DNA polymerase from the family B (PolB), granting these elements the category of the so-called “self-replicating” elements along with the abovementioned Polintons and Casposons. However, PolBs from pipolins do not need a DNA/RNA 3’-end or a protein as a primer to initiate the complementary strand synthesis, making pipolins a unique class of elements inside the MGE universe. Furthermore, the biochemical characterization of this polymerase named piPolB (from primer-independent) has shown that it has capacity of proofreading, strand displacement, and replication over both undamaged and damaged templates. All of these features plus the de-novo DNA synthesis suggest a possible role for the piPolB in pipolin replication and/or cell DNA damage tolerance.&nbsp;However, we could generate a <em>E. coli</em> strain harboring pipolin but lacking the piPolB, which indicates that piPolB is not essential for pipolin maintenance. This result raised questions about the biological role of piPolB, but also about the “self-replicating” GME. Are they replicating at all?<br>
Later on, we could characterize the pipolins in pathogenic <em>E. coli</em> strains, showing that they are highly flexible and diverse, with the piPolB and the integration site being the only features in common to all elements <span class="citation" data-cites="flament-simon2020">Flament-Simon et al. (<a href="#ref-flament-simon2020" role="doc-biblioref">2020</a>)</span>. We described this work also in this <a href="https://microbiologycommunity.nature.com/posts/pipolins-provided-an-opportunity-to-study-diversity-and-variability-of-self-replicative-genetic-mobile-elements-in-circulating-bacteria">post</a> in Nature Microbiology Community site.</p>
<p><img src="images/383903a6-985f-43a4-9ca7-27ebd1848073_large.avif" class="img-fluid"></p>
<p><em>E. coli</em> pipolins appear as integrated elements next to a specific tRNA gene, delimited by two direct repetitions (DRs) similar to phage atts, and encode for several distinct functions other than the piPolB. Among these different functions, we often find: tyrosine-recombinases (Y-rec), probably responsible for the excision and integration of the element; restriction-modification (RM) enzymes, known as defense systems against foreign DNA; and many other genes related to DNA mobilization and metabolism (integrases, Uracil-DNA glycosylases, excisionases). Surprisingly, no antimicrobial resistance gene was found in any pipolin except very few cases.<br>
</p>
</section>
<section id="explorepipolin" class="level2">
<h2 class="anchored" data-anchor-id="explorepipolin">ExplorePipolin</h2>
<p>The recent accessibility of high-throughput sequencing methods as part of surveillance programs of bacterial pathogens allows the genomic and metagenomic monitoring of the expansion of bacterial strain-specific markers, including virulence and AMR genes. However, these fast-evolving methods also generated a burden of data that must be throughout processed and analyzed <span class="citation" data-cites="mitchell2019">Mitchell and Simner (<a href="#ref-mitchell2019" role="doc-biblioref">2019</a>)</span>. This is particularly problematic regarding the study of dynamics and plasticity of MGEs, as they can range in size from very simple and small elements, such as insertion elements (IS), coding for only the transposase necessary for their relocation, to large prophages, transposons and plasmids, which can be tens or hundreds of kilobase pairs in length and also appear associated among themselves <span class="citation" data-cites="partridge2018">Partridge et al. (<a href="#ref-partridge2018" role="doc-biblioref">2018</a>)</span> <span class="citation" data-cites="durrant2020">Durrant et al. (<a href="#ref-durrant2020" role="doc-biblioref">2020</a>)</span> <span class="citation" data-cites="benler2021">Benler et al. (<a href="#ref-benler2021" role="doc-biblioref">2021</a>)</span>. Further, MGEs prediction and analysis are hindered by their great modularity and rapid evolution through gene acquisition and gene loss. Many pipelines designed for the analysis of MGEs are specialized and rely on the identification of hallmark genes, like plasmid replication proteins <span class="citation" data-cites="carattoli2020">Carattoli and Hasman (<a href="#ref-carattoli2020" role="doc-biblioref">2020</a>)</span>, relaxases <span class="citation" data-cites="alvarado2012">Alvarado, Garcillán-Barcia, and Cruz (<a href="#ref-alvarado2012" role="doc-biblioref">2012</a>)</span> or specific transposase or recombinases for integrative elements <span class="citation" data-cites="ross2021">Ross et al. (<a href="#ref-ross2021" role="doc-biblioref">2021</a>)</span> <span class="citation" data-cites="siguier2012">Siguier et al. (<a href="#ref-siguier2012" role="doc-biblioref">2012</a>)</span> <span class="citation" data-cites="moura2009">Moura et al. (<a href="#ref-moura2009" role="doc-biblioref">2009</a>)</span> <span class="citation" data-cites="cury2017">Cury, Touchon, and Rocha (<a href="#ref-cury2017" role="doc-biblioref">2017</a>)</span> <span class="citation" data-cites="cury2016">Cury et al. (<a href="#ref-cury2016" role="doc-biblioref">2016</a>)</span>. Some works have focused on the use of selected, high-quality reference genomes, but at the cost of diversity loss <span class="citation" data-cites="jiang2019">Jiang et al. (<a href="#ref-jiang2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>In order to facilitate the characterization of pipolins, we developed <a href="https://github.com/pipolinlab/ExplorePipolin">ExplorePipolin</a>, a Python-based pipeline that screens for the presence of the element and performs its reconstruction and annotation. The pipeline can be used on virtually any genome from diverse organisms and of diverse quality, obtaining the highest-scored possible structure and reconstructed out of different contigs if necessary. Then, predicted pipolin boundaries and pipolin encoded genes are subsequently annotated using a custom database, returning the standard file formats suitable for comparative genomics of this mobile element.</p>



</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-alvarado2012" class="csl-entry" role="doc-biblioentry">
Alvarado, Andrés, M. Pilar Garcillán-Barcia, and Fernando de la Cruz. 2012. <span>“A Degenerate Primer MOB Typing (DPMT) Method to Classify Gamma-Proteobacterial Plasmids in Clinical and Environmental Settings.”</span> <em>PLOS ONE</em> 7 (7): e40438. <a href="https://doi.org/10.1371/journal.pone.0040438">https://doi.org/10.1371/journal.pone.0040438</a>.
</div>
<div id="ref-benler2021" class="csl-entry" role="doc-biblioentry">
Benler, Sean, Guilhem Faure, Han Altae-Tran, Sergey Shmakov, Feng Zhang, and Eugene Koonin. 2021. <span>“Cargo Genes of Tn7-Like Transposons Comprise an Enormous Diversity of Defense Systems, Mobile Genetic Elements, and Antibiotic Resistance Genes.”</span> <em>mBio</em> 12 (6): e02938–21. <a href="https://doi.org/10.1128/mBio.02938-21">https://doi.org/10.1128/mBio.02938-21</a>.
</div>
<div id="ref-botelho2021" class="csl-entry" role="doc-biblioentry">
Botelho, João, and Hinrich Schulenburg. 2021. <span>“The Role of Integrative and Conjugative Elements in Antibiotic Resistance Evolution.”</span> <em>Trends in Microbiology</em>, Special Issue: Microbial Genetics: Stress Management, 29 (1): 8–18. <a href="https://doi.org/10.1016/j.tim.2020.05.011">https://doi.org/10.1016/j.tim.2020.05.011</a>.
</div>
<div id="ref-carattoli2020" class="csl-entry" role="doc-biblioentry">
Carattoli, Alessandra, and Henrik Hasman. 2020. <span>“PlasmidFinder and In Silico pMLST: Identification and Typing of Plasmid Replicons in Whole-Genome Sequencing (WGS).”</span> <em>Methods in Molecular Biology (Clifton, N.J.)</em> 2075: 285–94. <a href="https://doi.org/10.1007/978-1-4939-9877-7_20">https://doi.org/10.1007/978-1-4939-9877-7_20</a>.
</div>
<div id="ref-carvalho2020" class="csl-entry" role="doc-biblioentry">
Carvalho, Gabriel, David Fouchet, Gonché Danesh, Anne-Sophie Godeux, Maria-Halima Laaberki, Dominique Pontier, Xavier Charpentier, and Samuel Venner. 2020. <span>“Bacterial Transformation Buffers Environmental Fluctuations Through the Reversible Integration of Mobile Genetic Elements.”</span> <em>mBio</em> 11 (2): e02443–19. <a href="https://doi.org/10.1128/mBio.02443-19">https://doi.org/10.1128/mBio.02443-19</a>.
</div>
<div id="ref-cury2016" class="csl-entry" role="doc-biblioentry">
Cury, J., T. Jove, M. Touchon, B. Neron, and E. P. Rocha. 2016. <span>“Identification and Analysis of Integrons and Cassette Arrays in Bacterial Genomes.”</span> <em>Nucleic Acids Research</em> 44 (10): 4539–50. <a href="https://doi.org/10.1093/nar/gkw319">https://doi.org/10.1093/nar/gkw319</a>.
</div>
<div id="ref-cury2017" class="csl-entry" role="doc-biblioentry">
Cury, J., M. Touchon, and E. P. C. Rocha. 2017. <span>“Integrative and Conjugative Elements and Their Hosts: Composition, Distribution and Organization.”</span> <em>Nucleic Acids Research</em> 45 (15): 8943–56. <a href="https://doi.org/10.1093/nar/gkx607">https://doi.org/10.1093/nar/gkx607</a>.
</div>
<div id="ref-durrant2020" class="csl-entry" role="doc-biblioentry">
Durrant, Matthew G., Michelle M. Li, Benjamin A. Siranosian, Stephen B. Montgomery, and Ami S. Bhatt. 2020. <span>“A Bioinformatic Analysis of Integrative Mobile Genetic Elements Highlights Their Role in Bacterial Adaptation.”</span> <em>Cell Host &amp; Microbe</em> 27 (1): 140–153.e9. <a href="https://doi.org/10.1016/j.chom.2019.10.022">https://doi.org/10.1016/j.chom.2019.10.022</a>.
</div>
<div id="ref-firth2018" class="csl-entry" role="doc-biblioentry">
Firth, Neville, Slade O. Jensen, Stephen M. Kwong, Ronald A. Skurray, and Joshua P. Ramsay. 2018. <span>“Staphylococcal Plasmids, Transposable and Integrative Elements.”</span> <em>Microbiology Spectrum</em> 6 (6): 6.6.06. <a href="https://doi.org/10.1128/microbiolspec.GPP3-0030-2018">https://doi.org/10.1128/microbiolspec.GPP3-0030-2018</a>.
</div>
<div id="ref-flament-simon2020" class="csl-entry" role="doc-biblioentry">
Flament-Simon, Saskia-Camille, María de Toro, Liubov Chuprikova, Miguel Blanco, Juan Moreno-González, Margarita Salas, Jorge Blanco, and Modesto Redrejo-Rodríguez. 2020. <span>“High Diversity and Variability of Pipolins Among a Wide Range of Pathogenic Escherichia Coli Strains.”</span> <em>Scientific Reports</em> 10 (1). <a href="https://doi.org/10.1038/s41598-020-69356-6">https://doi.org/10.1038/s41598-020-69356-6</a>.
</div>
<div id="ref-frost2005" class="csl-entry" role="doc-biblioentry">
Frost, Laura S., Raphael Leplae, Anne O. Summers, and Ariane Toussaint. 2005. <span>“Mobile Genetic Elements: The Agents of Open Source Evolution.”</span> <em>Nature Reviews Microbiology</em> 3 (9): 722–32. <a href="https://doi.org/10.1038/nrmicro1235">https://doi.org/10.1038/nrmicro1235</a>.
</div>
<div id="ref-garcillan-barcia2009" class="csl-entry" role="doc-biblioentry">
Garcillan-Barcia, M. P., M. V. Francia, and F. de la Cruz. 2009. <span>“The Diversity of Conjugative Relaxases and Its Application in Plasmid Classification.”</span> <em>FEMS Microbiology Reviews</em> 33 (3): 657–87. <a href="https://doi.org/10.1111/j.1574-6976.2009.00168.x">https://doi.org/10.1111/j.1574-6976.2009.00168.x</a>.
</div>
<div id="ref-humphrey2021" class="csl-entry" role="doc-biblioentry">
Humphrey, Suzanne, Álvaro San Millán, Macarena Toll-Riera, John Connolly, Alejandra Flor-Duro, John Chen, Carles Ubeda, R. Craig MacLean, and José R. Penadés. 2021. <span>“Staphylococcal Phages and Pathogenicity Islands Drive Plasmid Evolution.”</span> <em>Nature Communications</em> 12 (1): 5845. <a href="https://doi.org/10.1038/s41467-021-26101-5">https://doi.org/10.1038/s41467-021-26101-5</a>.
</div>
<div id="ref-jiang2019" class="csl-entry" role="doc-biblioentry">
Jiang, Xiaofang, Andrew Brantley Hall, Ramnik J. Xavier, and Eric J. Alm. 2019. <span>“Comprehensive Analysis of Chromosomal Mobile Genetic Elements in the Gut Microbiome Reveals Phylum-Level Niche-Adaptive Gene Pools.”</span> <em>PLOS ONE</em> 14 (12): e0223680. <a href="https://doi.org/10.1371/journal.pone.0223680">https://doi.org/10.1371/journal.pone.0223680</a>.
</div>
<div id="ref-kapitonov2006" class="csl-entry" role="doc-biblioentry">
Kapitonov, V. V., and J. Jurka. 2006. <span>“Self-synthesizing DNA transposons in eukaryotes.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 103 (12): 4540–45. <a href="https://doi.org/10.1073/pnas.0600833103">https://doi.org/10.1073/pnas.0600833103</a>.
</div>
<div id="ref-krupovic2016" class="csl-entry" role="doc-biblioentry">
Krupovic, M., and E. V. Koonin. 2016. <span>“Self-Synthesizing Transposons: Unexpected Key Players in the Evolution of Viruses and Defense Systems.”</span> <em>Current Opinion in Microbiology</em> 31 (June): 25–33. <a href="https://doi.org/10.1016/j.mib.2016.01.006">https://doi.org/10.1016/j.mib.2016.01.006</a>.
</div>
<div id="ref-krupovic2014" class="csl-entry" role="doc-biblioentry">
Krupovic, M., K. S. Makarova, P. Forterre, D. Prangishvili, and E. V. Koonin. 2014. <span>“Casposons: A New Superfamily of Self-Synthesizing DNA Transposons at the Origin of Prokaryotic CRISPR-Cas Immunity.”</span> <em>BMC Biol</em> 12 (1): 36. <a href="https://doi.org/10.1186/1741-7007-12-36">https://doi.org/10.1186/1741-7007-12-36</a>.
</div>
<div id="ref-magiorakos2012" class="csl-entry" role="doc-biblioentry">
Magiorakos, A. P., A. Srinivasan, R. B. Carey, Y. Carmeli, M. E. Falagas, C. G. Giske, S. Harbarth, et al. 2012. <span>“Multidrug-Resistant, Extensively Drug-Resistant and Pandrug-Resistant Bacteria: An International Expert Proposal for Interim Standard Definitions for Acquired Resistance.”</span> <em>Clin Microbiol Infect</em> 18 (3): 268–81. <a href="https://doi.org/10.1111/j.1469-0691.2011.03570.x">https://doi.org/10.1111/j.1469-0691.2011.03570.x</a>.
</div>
<div id="ref-mitchell2019" class="csl-entry" role="doc-biblioentry">
Mitchell, Stephanie L., and Patricia J. Simner. 2019. <span>“Next-Generation Sequencing in Clinical Microbiology: Are We There Yet?”</span> <em>Clinics in Laboratory Medicine</em> 39 (3): 405–18. <a href="https://doi.org/10.1016/j.cll.2019.05.003">https://doi.org/10.1016/j.cll.2019.05.003</a>.
</div>
<div id="ref-moura2009" class="csl-entry" role="doc-biblioentry">
Moura, A., M. Soares, C. Pereira, N. Leitao, I. Henriques, and A. Correia. 2009. <span>“INTEGRALL: A Database and Search Engine for Integrons, Integrases and Gene Cassettes.”</span> <em>Bioinformatics</em> 25 (8): 1096–98. <a href="https://doi.org/10.1093/bioinformatics/btp105">https://doi.org/10.1093/bioinformatics/btp105</a>.
</div>
<div id="ref-partridge2018" class="csl-entry" role="doc-biblioentry">
Partridge, S. R., S. M. Kwong, N. Firth, and S. O. Jensen. 2018. <span>“Mobile Genetic Elements Associated with Antimicrobial Resistance.”</span> <em>Clinical Microbiology Reviews</em> 31 (4). <a href="https://doi.org/10.1128/CMR.00088-17">https://doi.org/10.1128/CMR.00088-17</a>.
</div>
<div id="ref-redrejo-rodríguez2017" class="csl-entry" role="doc-biblioentry">
Redrejo-Rodríguez, M., C. D. Ordóñez, M. Berjón-Otero, J. Moreno-González, C. Aparicio-Maldonado, P. Forterre, M. Salas, and M. Krupovic. 2017. <span>“Primer-Independent DNA Synthesis by a Family b DNA Polymerase from Self-Replicating Mobile Genetic Elements.”</span> <em>Cell Rep</em> 21 (6): 1574–87. <a href="https://doi.org/10.1016/j.celrep.2017.10.039">https://doi.org/10.1016/j.celrep.2017.10.039</a>.
</div>
<div id="ref-ross2021" class="csl-entry" role="doc-biblioentry">
Ross, Karen, Alessandro M. Varani, Erik Snesrud, Hongzhan Huang, Danillo Oliveira Alvarenga, Jian Zhang, Cathy Wu, Patrick McGann, and Mick Chandler. 2021. <span>“TnCentral: a Prokaryotic Transposable Element Database and Web Portal for Transposon Analysis.”</span> <em>mBio</em> 12 (5): e0206021. <a href="https://doi.org/10.1128/mBio.02060-21">https://doi.org/10.1128/mBio.02060-21</a>.
</div>
<div id="ref-siguier2012" class="csl-entry" role="doc-biblioentry">
Siguier, P., A. Varani, J. Perochon, and M. Chandler. 2012. <span>“Exploring Bacterial Insertion Sequences with ISfinder: Objectives, Uses, and Future Developments.”</span> <em>Methods in Molecular Biology</em> 859: 91–103. <a href="https://doi.org/10.1007/978-1-61779-603-6_5">https://doi.org/10.1007/978-1-61779-603-6_5</a>.
</div>
<div id="ref-smillie2010" class="csl-entry" role="doc-biblioentry">
Smillie, Chris, M. Pilar Garcillán-Barcia, M. Victoria Francia, Eduardo P. C. Rocha, and Fernando de la Cruz. 2010. <span>“Mobility of Plasmids.”</span> <em>Microbiology and Molecular Biology Reviews</em> 74 (3): 434–52. <a href="https://doi.org/10.1128/MMBR.00020-10">https://doi.org/10.1128/MMBR.00020-10</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
    var links = window.document.querySelectorAll('a:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
          // default icon
          link.classList.add("external");
      }
    }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">Copyright 2023, Modesto Redrejo Rodríguez</div>   
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/pipolinlab">
      <i class="bi bi-github" role="img">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/RnR_Lab">
      <i class="bi bi-twitter" role="img">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>



<script src="site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>